iai_boxy_base_2/robots/base.xacro

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0"?>

```

Comment:
You renamed base.URDF to base.xacro? Why did you do that and why is the new file shorter? Other robot urdfs are using this one as well, so the risk of breaking the other urdfs has to be worth the gain from changing it.

Reply:
It is shorter because I used an auto-format, the results are still the same. The different is now we have an option is to use or not to use odom.
So instead of writing  
<xacro:base/>
we write
<xacro:base odom="True"/>